mozci/console/commands/push.py

```diff
@@ -363,10 +395,40 @@ def log_pushes(self, status, backedout):
             ]
         )
         verb = "were" if backedout else "weren't"
-        self.line(
-            f"{nb} out of {len(self.pushes)} pushes {verb} backed-out by a sheriff and were classified as {status.name}."
+        line = f"{nb} out of {len(self.pushes)} pushes {verb} backed-out by a sheriff and were classified as {status.name}."
+        self.line(line)
+
+        return line
+
+    def send_emails(self, total, stats, error_line):
+        notify = taskcluster.Notify(get_taskcluster_options())
+
+        today = datetime.datetime.strftime(datetime.datetime.now(), "%Y-%m-%d")
+        subject = f"Mozci | classify-eval report generated the {today}"
+
+        stats = "\n".join([f"- {stat}" for stat in stats])
+        content = EMAIL_CONTENT.format(
+            today=today,
+            total=total,
+            error_line=f"**{error_line}**" if error_line else "",
+            stats=stats,
         )
 
+        emails = config.get("emails", [])
```

Comment:
FYI this works as expected with a local config in `~/.config/mozci/config.toml` with the content:

```toml
[mozci]
emails = ["abadie@teklia.com", "marco@mozilla.com"]

```